BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/DocumentPages/Error/DocumentPagesErrorView.swift

```diff
@@ -0,0 +1,197 @@
+        button.titleLabel?.font = configuration.textStyleFonts[.bodyBold]
+        button.configure(with: configuration.primaryButtonConfiguration)
+        button.addTarget(self, action: #selector(didPressButton), for: .touchUpInside)
+    }
+
+    @objc private func didPressButton() {
+        buttonAction?()
+    }
+
+    private func configureConstraints() {
+        configureHeaderConstraints()
+        configureScrollViewConstraints()
+        configureButtonViewConstraints()
+        configureErrorContentConstraints()
+    }
+
+    private func configureHeaderConstraints() {
+        if UIDevice.current.isIpad {
+            NSLayoutConstraint.activate([
+                errorHeader.headerStack.widthAnchor.constraint(equalTo: widthAnchor,
+                                                               multiplier: Constants.iPadWidthMultiplier),
+                errorHeader.headerStack.centerXAnchor.constraint(equalTo: centerXAnchor)
+            ])
+        } else {
+            NSLayoutConstraint.activate([
+                errorHeader.headerStack.leadingAnchor.constraint(equalTo: leadingAnchor,
+                                                                 constant: Constants.stackViewLeadingPadding),
+                errorHeader.headerStack.trailingAnchor.constraint(equalTo: trailingAnchor,
+                                                                  constant: -Constants.sidePadding)
+            ])
+        }
+        errorHeader.setContentHuggingPriority(UILayoutPriority.defaultHigh, for: .vertical)
+        errorHeader.setContentCompressionResistancePriority(.defaultLow, for: .vertical)
+
+        NSLayoutConstraint.activate([
+            errorHeader.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor),
+            errorHeader.leadingAnchor.constraint(equalTo: leadingAnchor),
+            errorHeader.trailingAnchor.constraint(equalTo: trailingAnchor),
+            errorHeader.heightAnchor.constraint(greaterThanOrEqualToConstant: Constants.errorHeaderMinHeight),
+            errorHeader.heightAnchor.constraint(lessThanOrEqualTo: heightAnchor,
+                                                multiplier: Constants.errorHeaderHeightMultiplier)
+        ])
+    }
+
+    private func configureScrollViewConstraints() {
+        NSLayoutConstraint.activate([
+            scrollView.topAnchor.constraint(equalTo: errorHeader.bottomAnchor),
+            scrollView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            scrollView.bottomAnchor.constraint(equalTo: button.topAnchor)
+        ])
+    }
+
+    private func configureButtonViewConstraints() {
+        NSLayoutConstraint.activate([
+            button.heightAnchor.constraint(greaterThanOrEqualToConstant: Constants.singleButtonHeight),
+            button.bottomAnchor.constraint(equalTo: safeAreaLayoutGuide.bottomAnchor, constant: -GiniMargins.margin)
```

Comment:
[clarification] This is the first time I've seen GiniMargins. I suggest having local constants for this since we have not used this struct until now 🫤 .